estimator/lwe_dual.py

```diff
@@ -172,6 +185,56 @@ def cost(
         # just guess different components of the secret
         return cost.repeat(times=rep, select={"m": False})
 
+    @staticmethod
+    def gj21_solver(params, success_probability, t=0):
```

Comment:
should we call this "FFT" solver or so? Say, someone improves it, ideally we'd update this function rather than adding a new one (?)